schema-engine/sql-migration-tests/tests/schema_push/mod.rs

```diff
@@ -489,5 +489,8 @@ model m1 {
     }))
     .unwrap_err();
 
-    assert!(&err.to_string().starts_with(r#"The `mysql` database is a system database, it should not be altered with prisma migrate. Please connect to another database."#));
+    let err_msg = err.message().unwrap();
+    let expected = expect!["multiSchema migrations and introspection are not implemented on MySQL yet"];
```

Comment:
This is actually the correct error, explicitly added in https://github.com/prisma/prisma-engines/commit/574e45f3fd7c55b8ffcea000812329a79e7f2a22 (although the snapshot test caught a different error even in the same commit).

It used to be superseded with a different error that doesn't sound super relevant due to the `ensure_connection_validity()` call in `schema_push` (that was necessary to support `db push` with CockroachDB via `postgresql` connector), which these changes remove.